optimizely/config/polling_manager.go

```diff
@@ -53,42 +53,40 @@ type PollingProjectConfigManager struct {
 	exeCtx utils.ExecutionCtx // context used for execution control
 }
 
-func (cm *PollingProjectConfigManager) activate(initialPayload []byte, init bool) {
-
-	update := func() {
-		var e error
-		var code int
-		var payload []byte
-		if init && len(initialPayload) > 0 {
-			payload = initialPayload
-		} else {
-			payload, code, e = cm.requester.Get()
-
-			if e != nil {
-				cmLogger.Error(fmt.Sprintf("request returned with http code=%d", code), e)
-			}
-		}
-
-		projectConfig, err := datafileprojectconfig.NewDatafileProjectConfig(payload)
-		if err != nil {
-			cmLogger.Error("failed to create project config", err)
+func (cm *PollingProjectConfigManager) Sync(datafile []byte) {
```

Comment:
please add a comment, otherwise linters will complain 
comment has to be like // SyncConfig .......